setup.py

```diff
@@ -94,9 +95,9 @@
         "ijson>=3.1.4,<4",
         "importlib-metadata",
         "ipython>=8.5.0,<9",
-        "packaging>=20.9,<21",
+        "packaging>=21,<22",
         "pandas>=1.3.0,<2",
-        "pluggy>=1.0.0,<2",
+        "pluggy>=0.12,<2",
         "pydantic>=1.9.2,<2",
         "PyGithub>=1.54,<2",
         "pytest>=6.0,<8.0",
```

Comment:
Let's try bumping pytest to a version that does require pluggy < 1.0

Reply:
you mean > 1.0 right? otherwise we cant use pytest 7 at all, and pin it to 6 or so which is not good at all

Reply:
and >1.0 it. doesnt exist

Reply:
do you have a reason to pin pluggy >1.00 <2? i mean it is used by pytest and pytest newer versions using it with >0.12 <2

Reply:
It is the odd that you can't use pluggy 1.0.0 with pytest 7; you must be using Poetry or something?
However, I see that pytest supports this same range. Thanks for pointing that out! We will go with it.